ParameterTrendPercentages/ParameterTrendPercentages_1/ParameterTrendPercentages_1.cs

```diff
@@ -194,9 +191,11 @@ private void RetrieveTrendRecords()
     {
         TrendingType trendingType = TrendingType.Realtime;
 
+        // Throw exception if time range is incorrect.
         if (_start >= _end)
```

Comment:
Same here, this could have been validated during the processing of the arguments. I believe this will now also throw when leaving the arguments empty as both will default to the same value.